torch/distributed/optim/apply_optimizer_in_backward.py

```diff
@@ -88,3 +88,26 @@ def optimizer_hook(*_unused) -> None:
 
     for param in params:
         _apply_optimizer_in_backward_to_param(param)
+
+
+def _get_optimizers_in_backward(model: torch.nn.Module) -> List[torch.optim.Optimizer]:
```

Comment:
Do you think `_get_in_backward_optimizers()` would read better? Does TorchRec have an API like this; if so, what is it called?